docs/src/components/Nav/NavLinkNext.js

```diff
@@ -0,0 +1,23 @@
+import pt from 'prop-types';
+import Link from 'next/link';
+
+const propTypes = {
+  children: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  isActive: pt.bool,
+};
+
+const NavLinkNext = ({ children, isActive, ...rest }) => {
```

Comment:
```suggestion
// This component is needed as a compatibility layer inside of AdminStructure.Nav due to the odd way that
// Link works inside of `next/link` where it does not render an element itself. This component allows us to 
// maintain a structure that is consistent with other link components, and direct props into the expected location.
const NavLinkNext = ({ children, isActive, ...rest }) => {
```

Reply:
I think this note will be helpful for our future selves if someone comes along and sees this component later. I'd recommend just committing this suggestion directly to make things easier.